scipy/signal/_signaltools.py

```diff
@@ -1588,11 +1588,12 @@ def medfilt(volume, kernel_size=None):
     return result
 
 
-def wiener(im, mysize=None, noise=None):
+def wiener(im, mysize=None, noise=None, boundary='fill', fillvalue=0):
```

Comment:
(Correct me if I'm wrong maintainers but) I think we'd want to add the new parameters as keyword-only

```suggestion
def wiener(im, mysize=None, noise=None, *, boundary='fill', fillvalue=0):
```

Reply:
Is this somewhere in the guidelines for contributions? No problem with adding that.

Reply:
https://docs.scipy.org/doc/scipy/dev/missing-bits.html#missing-bits

Reply:
It may be the case that we want to move the `*` forward straight after this but that's a deprecation decision.